pages/portfolio.html

```diff
@@ -5,8 +5,43 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" type="text/css" rel="stylesheet"/>
 </head>
-<body>
-  
+<body class="grid">
+  <header class="header header-image main-header">
+    <ul>
+      <li class="main-header-element">Mariah Rosman</li>
+      <li>she/her/hers</li>
+    </ul>
+  </header>
+  <nav>
+    <ul class="main-nav">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About Me</a></li>
+      <li><a href="portfolio.html">My Portfolio</a></li>
+      <li><img class="nav-image" src="https://scontent-sea1-1.xx.fbcdn.net/v/t1.6435-9/78495079_10157062903949682_1158759696601972736_n.jpg?_nc_cat=111&ccb=1-5&_nc_sid=09cbfe&_nc_ohc=3S6JI1kA_NUAX9xhhba&tn=WQy3tMs_McsdNvDY&_nc_ht=scontent-sea1-1.xx&oh=159cf5bdb2247842637c112d67dd3b50&oe=61C850B3"
+        alt="Mariah in roller derby gear smiling to her left"></li>
+    </ul>
+  </nav>
+  <p>
+    <ul class="portfolio">
+      <li><a href="https://github.com/mrosman7/personal-portfolio-site">Personal Portfolio Site</a></li>
+      <li><a href="https://github.com/mrosman7/task-list-api">Task List API</a></li>
+      <li><a href="https://github.com/mrosman7/hello-books-api">Hello Books API</a></li>
+      <li><a href="https://github.com/mrosman7/swap-meet">Swap Meet</a></li>
+      <li><a href="https://github.com/mrosman7/viewing-party">Viewing Party</a></li>
+  </ul></p>
```

Comment:
The `<p>` tag can not contain block level elements, and will automatically close if followed by a block level element (such as `<ul>`) before the `</p>` tag, which causes the closing tag on line 33 to be unmatched.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p.  For this section I'd recommend using something more semantically meaningful, such as the `<article>` or `<section>` tag, both of which can contain block level elements.